egs/fuss/baseline/local/prepare_data.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Which part of this script have Google Copyright?

Reply:
Each process in the pipeline is copyrighted (data downloading, dataset creation and data augmentation). This script unifies and simplifies all the processes of the official FUSS repository related to the download and creation of the dataset.